dev-docs/source/Testing/MuonAnalysis_test_guides/Muon_ALC.rst

```diff
@@ -0,0 +1,88 @@
+.. _Muon_ALC-ref:
+
+Muon Unscripted Testing: ALC
+============================
+
+.. contents:: Table of Contents
+   :local:
+
+Introduction
+------------
+
+These tests looks at the ALC interface.
+This interface loads a series of runs and automatically calculate the asymmetry.
+The asymmetry calculation is determined by the settings on the load page and are for a pair.
+It is often used to find peaks in the asymmetry as a function of a sample log (e.g. temperature).
+
+-----------------------------------------------
+
+.. _ALC_basic:
+
+ALC peak fitting
+----------------
+
+**Time required 10 - 15 minutes**
+
+- Open **ALC** (*Interfaces* > *Muon* > *ALC*)
+- Change *Instrument* to **EMU**
+- In the loading bar enter ``81061-142``
```

Comment:
Ditto on cutting this down to 2-3 runs if possible to make it easier on testers grabbing the data, especially on slower connections

Reply:
Its needed for the analysis later on in the run